src/parser/demonhunter/havoc/modules/talents/BlindFury.js

```diff
@@ -70,11 +70,12 @@ class BlindFury extends Analyzer{
       <TalentStatisticBox
         talent={SPELLS.BLIND_FURY_TALENT.id}
         position={STATISTIC_ORDER.OPTIONAL(6)}
-        value={
-          <>
+        value={(
+<>
```

Comment:
this was autofixed by eslint. I'd have fixed the indentation but I couldn't be bothered